tests/TestCase/Model/Behavior/UploadBehaviorTest.php

```diff
@@ -28,7 +28,8 @@ public function setup()
                 'error' => UPLOAD_ERR_OK,
                 'size' => 1,
                 'type' => 'text',
-                'keepFilesOnDelete' => false
+                'keepFilesOnDelete' => false,
+                'deleteCallback' => false
```

Comment:
My personal preference would be `null` here to demonstrate a 'nothing' value, rather than a boolean.